roles/libvirt_manager/tasks/start_manage_vms.yml

```diff
@@ -67,7 +67,7 @@
     extracted_ip: "{{ (vm_ip.stdout.split())[3] | ansible.utils.ipaddr('address') }}"
     identity_file: >-
       {{
-        cifmw_reproducer_basedir ~ '/artifacts/cifmw_ocp_access_key' if vm_type == 'ocp' else
+        cifmw_libvirt_manager_basedir ~ '/artifacts/cifmw_ocp_access_key' if vm_type == 'ocp' else
```

Comment:
right - it was called from the reproducer until now, so it wasn't failing. Values should be the same at this point.